backend/cron/service.go

```diff
@@ -78,6 +78,7 @@ func run(ctx context.Context, verbClient CallClient, changes chan *ftlv1.PullSch
 	logger := log.FromContext(ctx).Scope("cron")
 	// Map of cron jobs for each module.
 	cronJobs := map[string][]cronJob{}
+	currentDeploymentForModule := map[string]string{}
```

Comment:
What I don't understand is that it looks like keys are never removed from this map?

Reply:
oops, that was an oversight, fixed. It would not have caused any actual issues in practice, unless you added and removed a huge number of modules.